core/src/test/java/tech/tablesaw/io/fixed/FixedWidthReaderTest.java

```diff
@@ -79,7 +82,7 @@ public void testWithColumnSKIP() throws Exception {
 
     assertEquals(4, table.columnCount());
 
-    String[] expected = new String[]{"Year", "Make", "Model", "Price"};
+    String[] expected = new String[] {"Year", "Make", "Model", "Price"};
```

Comment:
if you're going to clean these up, why not just 
```
    String[] expected = {"Year", "Make", "Model", "Price"};
```

Reply:
I don't understand what do you mean.
The change was from the formatted plugin. I ignored changes from not touched files but included the ones from changed files.

Reply:
You introduced a space between new String[] and the array literal. 

The "new String[]" part is optional, so why not just remove it?

Reply:
Just a suggestion. Seems a little cleaner to me to leave it out entirely.